hack/prune-junit-xml/prunexml.go

```diff
@@ -0,0 +1,224 @@
+/*
+Copyright 2022 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Copyright 2022 The Kubernetes Authors.
+Started from https://github.com/dims/kubernetes/blob/d20df79545a41e499082f3786f9953d65eee450d/cmd/prune-junit-xml/prunexml.go
+*/
+
+package main
+
```

Comment:
suggestion: I think this file would benefit a lot from having a comment at the top explaining what it's for and why we need it. I can see we _do_ need it, for filtering out fuzz tests - just it'd be good to have that in this file, for people in the future to see at a glance why we have this!